packages/tooltip/src/Tooltip.ts

```diff
@@ -246,6 +247,29 @@ export class Tooltip extends SpectrumElement {
         return triggerElement;
     }
 
+    @state()
+    private dependenciesLoaded = false;
+    private dependenciesToLoad: Record<string, boolean> = {};
+
+    private trackDependency(dependency: string, flag?: boolean): void {
```

Comment:
Is it worth abstracting this into a helper method since it exists in Picker, too? Do we foresee needing this in other places in the future?

Reply:
Yes. I'll make an issue to track. We could at the very least use this approach in Menu Item, but possibly some others, too.

Reply:
https://github.com/adobe/spectrum-web-components/issues/3900